py/desisim/scripts/quickquasars.py

```diff
@@ -523,7 +563,19 @@ def simulate_one_healpix(ifilename,args,model,obsconditions,decam_and_wise_filte
         zbest = Table(np.zeros(nqso, dtype=columns))
         zbest["CHI2"][:]      = 0.
         zbest["Z"]            = metadata['Z']
-        zbest["ZERR"][:]      = 0.
+        zbest["ZERR"][:]      = 0. 
+        if args.sigma_kms_rsd or args.sigma_kms_zfit:
+           sigma_rsd,sigma_fit = 0.,0.
+           if args.sigma_kms_rsd:
+              sigma_rsd=args.sigma_kms_rsd
+              log.info("Added RSD error with sigma {} to zbest".format(sigma_rsd))
+           if args.sigma_kms_zfit:
+              sigma_fit=args.sigma_kms_zfit
+              log.info("Added zfit error with sigma {} to zbest".format(sigma_fit))
+           sigma=np.sqrt(sigma_rsd*sigma_rsd+sigma_fit*sigma_fit)
+           dz=(sigma/299792.458)*(1.+metadata['Z'])*np.random.normal(0,1,nqso)
+           zbest["Z"]+=dz
+           zbest["ZERR"]+=dz
```

Comment:
OK. I was also unsure if also update this field zbest["ZERR"], as I did, or if it would be better to be other random number with around dz? Which according what you just said dz will be only due to dz_fit.